app/Http/Requests/Category/CategoryStoreRequest.php

```diff
@@ -24,8 +24,8 @@ public function authorize()
     public function rules()
     {
         return [
-            'name' => ['required'],
-            'description' => ['required'],
+            'name' => ['required','max:30'],
```

Comment:
Always have space after comma
```suggestion
'name' => ['required', 'max:30'],
```

Reply:
This is resolved.Thank you